org.OpenT2T.Onboarding.WinkHub/v2/winkonboarding.js

```diff
@@ -0,0 +1,140 @@
+/* jshint esversion: 6 */
+/* jshint node: true */
+
+'use strict';
+var https = require('https');
+var inquirer = require('inquirer');
+var q = require('q');
+var hubDevice = require('./WinkHub');
+var accessTokenInfo = require('../../org.OpenT2T.Onboarding.Hub/opent2tonboarding');
+
+// module exports, implementing the schema
+module.exports = {
+
+    //
+    // question: Do we need to remove validate function?
+    // This is tied with 'inquirer' syntax, which helps validate at commandline as well.
+    //
+    onboardingFlow : [
+        {
+            action: 'get-user-input',

```

Comment:
I see two different data sets in here, one set which the *developer* provides (client_id, client_secret) and the other which the *end user* provides (username, password).  They have to be separately acquired.  The developer ones will likely be passed in like the nest sample and the end user ones like the osram ones does.  So I expect to see two items in the onboarding flow array, 'get-use-input' and 'get-developer-input'